language-plutus-core/generators/Language/PlutusCore/Generators/Internal/Dependent.hs

```diff
@@ -45,11 +45,11 @@ instance GShow uni => GEq (AsKnownType uni) where
         guard $ display @String a == display b
         Just $ unsafeCoerce Refl
```

Comment:
I've attempted to resolve this once again, to no avail. `Typeable` stuff is not a good option (whether it's `Data.Typeable` or `Type.Reflection`) as it leaks into important instances and I don't think we need that noise there. Such a stupid problem. I'll think more on it.